<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" tests="10" skipped="0" failures="0" errors="0" timestamp="2023-06-30T19:35:30" hostname="fv-az1104-758" time="0.363">
  <properties/>
  <testcase name="testEmptyParameter" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.233"/>
  <testcase name="testUserPasswordException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.102"/>
  <testcase name="testgetUsersWithLoginAs" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.01"/>
  <testcase name="testAuthException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testgetUsersWithoutLoginAs" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.003"/>
  <testcase name="testNoSuchUserException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.001"/>
  <testcase name="testRequiredLayoutException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testUserActiveException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.003"/>
  <testcase name="testWrongParameter" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testUserEmailAddressException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.003"/>
  <system-out><![CDATA[ErrorEntity{errorCode='your-account-is-not-active', message='your-account-is-not-active', fieldName='null'}
]]></system-out>
  <system-err><![CDATA[Jun 30, 2023 7:35:30 PM com.dotcms.repackage.org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 4.3.2.Final
]]></system-err>
</testsuite>
